src/main/groovy/nl/javadude/gradle/plugins/license/LicenseResolver.groovy

```diff
@@ -189,6 +190,18 @@ class LicenseResolver {
         fileDependencies
     }
 
+    boolean isDependencyIncluded(String depName, List<String> patternsToIgnore){
+        for(String pattern: patternsToIgnore){
+            if(depName.equals(pattern)){
+                return false;
+            }
+            if(Pattern.compile(pattern).matcher(depName).matches()){
```

Comment:
Let's compile the Patterns only once. With a lot of dependencies * a lot of patterns, this becomes painful.

Reply:
Sure thing. Will do!